File: commons/dto/src/main/java/net/blueshell/dto/RedirectDTO.java
package net.blueshell.dto;


import lombok.Data;
import lombok.EqualsAndHashCode;
import net.blueshell.enums.PlatformType;

import java.sql.Timestamp;

@Data
@EqualsAndHashCode(callSuper = true)
public class RedirectDTO extends BaseDTO {
    private String id;
    private Timestamp created;
    private TelemetryDTO telemetry;
}

File: api/src/main/kotlin/net/blueshell/api/telemetry/web/dto/RedirectDTO.kt
package net.blueshell.api.telemetry.web.dto

import io.swagger.v3.oas.annotations.media.Schema
import net.blueshell.api.shared.dto.AuditedAutoIdDTO

@Schema(name = "Redirect")
data class RedirectDTO(
    var telemetry: TelemetryDTO? = null
) : AuditedAutoIdDTO()
